### DISCUSSION TITLE
Whta is your favorite model of bike/What model do you ride?

### DISCUSSION THREAD

**Discussion by asuisan** [NAIST-SE/AttractingContributors#1080] [2024 Attraction Contributions]
For example, Honda Cub, Kawasaki Ninja, or Yamaha YZF.
Feel free to post about the bike you are currently riding or amy bile you like.
pullreq url : [here!](https://github.com/asuisan/AttractingContributors)

**Comment by 3207-Rhims** [comment]
https://github.com/3207-Rhims/AttractingContributors-Movie , would you be kind to contribute in my project. I made a PR fot you,

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon
